paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTabLayoutUI.kt

```diff
@@ -378,6 +379,10 @@ private fun SavedPaymentMethodTab(
             testTag = SAVED_PAYMENT_OPTION_TEST_TAG
             selected = isSelected
             text = AnnotatedString(labelText)
+
+            if (!isEnabled) {
+                disabled()
```

Comment:
Can you add a UI unit test for this?

Reply:
Looks like we have tests [here](https://github.com/stripe/stripe-android/blob/2e8ebfa2502c767cfa58789d40375f0a2d8265ed/paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PaymentOptionsTest.kt#L19) that could be added to test this bit.

That test probably needs a rename (my bad), but not important for this change, I can follow up with the rename if you want.